Lib/test/test_sqlite3/__init__.py

```diff
@@ -0,0 +1,22 @@
+import test.support
+from test.support import import_helper
+from test.support import load_package_tests
+
+# Skip test if _sqlite3 module not installed
+import_helper.import_module('_sqlite3')
+
+import unittest
+import os
+import sqlite3
+
+def load_tests(*args):
+    pkg_dir = os.path.dirname(__file__)
+    return load_package_tests(pkg_dir, *args)
```

Comment:
Where does this get used (i.e. no docstring as to why this needs to be explicitly specified)?

Reply:
It's because of the [load tests protocol](https://docs.python.org/3/library/unittest.html#load-tests-protocol). I'll add a docstring.